update_aimpoint_data.py

```diff
@@ -31,10 +32,25 @@ def get_opt():
     return parser.parse_args()
 
 
-def get_asol(obsid, asol_files, dt, cols=('time', 'dy', 'dz', 'dtheta')):
+def get_asol(obsid, asol_files, dt):
     logger.info('Reading...\n{}'.format('\n'.join(asol_files)))
-    asols = [Table.read(asol_file)[cols] for asol_file in asol_files]
+    asols = [Table.read(asol_file) for asol_file in asol_files]
+    # Check to see if they all have raw columns
```

Comment:
Need some whitespace before the comments in this function and should probably also mention the DS10.8.3 differences.